diff --git a/node_modules/telegraf-session-s3/src/session.js b/node_modules/telegraf-session-s3/src/session.js
index 48fad91..0d80709 100644
--- a/node_modules/telegraf-session-s3/src/session.js
+++ b/node_modules/telegraf-session-s3/src/session.js
@@ -1,6 +1,6 @@
 const debug = require('debug')('telegraf:session-s3');
 const AWS = require('aws-sdk');
-const s3 = new AWS.S3();
+const s3 = new AWS.S3({ region: process.env.AWS_REGION, endpoint: process.env.AWS_S3_ENDPOINT });
 
 function getS3Object(bucket, file) {
     return new Promise((resolve, reject) => {
